pennylane/devices/__init__.py

```diff
@@ -32,3 +32,5 @@
 """
 from .default_qubit import DefaultQubit
 from .default_gaussian import DefaultGaussian
+from .default_qubit_tf import DefaultQubitTF
+from .default_qubit_autograd import DefaultQubitAutograd
```

Comment:
Did we just forget to add these, or was there a reason?

Reply:
No this is intentional! `DefaultQubit` and `DefaultGaussian` have always been here, but not including `DefaultQubitTF` and `DefaultQubitAutograd` is deliberate. Including them here causes an 'import chain' on `import pennylane`, leading to an `import tensorflow` and an `import autograd` statement.

TensorFlow is an optional requirement, so we don't want this to happen (we only want `default.qubit.tf` to be imported when the user explicitly loads it)

Reply:
It might be good to either 

1. leave a comment here explaining that devices with optional requirements are not imported here, or

2. remove all device imports (but this means updating import paths from
  ```python
   from pennylane.devices import DefaultQubit
  ```
  to 
  ```python
  from pennylane.devices.default_qubit import DefaultQubit
  ```